src/SeoToolkit.Umbraco.Redirects/wwwroot/backoffice/Redirects/list.controller.js

```diff
@@ -98,6 +106,22 @@
             goToPage(1);
         }
 
+        function openImportDialog(model) {
+            var redirectDialogOptions = {
+                title: vm.dialogLabel,
+                view: "/App_Plugins/SeoToolkit/Redirects/Dialogs/import.html",
+                size: "small",
+                close: function () {
+                    editorService.close();
```

Comment:
I think it might be good to call `loadItems` here so that the user can immediately see the urls being added after closing the dialog instead of having to refresh the page